### ISSUE TITLE
It seems the filename checks are case sensitive, at least in Meikyuu

### ISSUE THREAD

**Issue by senpaizuri3** [trigger-death/GrisaiaExtractor#6]
I'm going through the files in Meikyuu, changing the sorting to how I personally would have the folder structure, and I'm noticing that `SD` is not caught in `sd` things, for chibi.

**Comment by trigger-segfault** [comment]
Yeah, I should probably update certain things to be case insensitive, although it's not as easy as it sounds because there are still categorization flags that are case sensitive. Where are these SD chibis ending up?

Also I should mention, as I write the new framework for the Grisaia Sprite Viewer, I'll probably spend a lot more time cleaning up the sprite directory sorting system. Maybe even make it customizable via scripts.

**Comment by senpaizuri3** [comment]
The SD chibis are ending up in the correct character's folder from what I remember, but in the CG subfolder and not the Chibi subfolder.

Edit: Wait... What are the categorizations that need to stay case sensitive? The only things I can remember that I wanted case sensitive from using exkifint on my own before using this was to have `*L.bmp` files and `*_m.bmp` in their own separate folders (since they are mostly like duplicates).
